packages/chevrotain/.mocharc.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
   recursive: true,
-  require: ["./test/test.config", "source-map-support/register"],
+  require: [
+    "./test/test.config",
+    "source-map-support/register",
+    "ts-node/register"
+  ],
   reporter: "spec",
-  spec: "./lib/test/**/*spec.js"
+  spec: "./test/**/*spec.ts"
```

Comment:
The "build tests and source first, then run built tests" seemed a bit awkward. Also it had problems with wrong source lines in stack traces.

This runs the .ts spec files directly on the .ts source. This should have the advantage that the coverage report is more useful as it applies to the the .ts source. Also fix-coverage-report shouldn't be necessary anymore (?)